forms/common-themes/src/main/resources/theme/base/admin/resources/partials/required-actions.html

```diff
@@ -20,8 +20,8 @@
         <tbody>
         <tr ng-repeat="requiredAction in requiredActions" data-ng-show="requiredActions.length > 0">
             <td>{{requiredAction.name}}</td>
-            <td><input type="checkbox" ng-model="requiredAction.enabled" ng-change="updateRequiredAction(requiredAction)"></td>
-            <td><input type="checkbox" ng-model="requiredAction.defaultAction" ng-change="updateRequiredAction(requiredAction)"></td>
+            <td><input type="checkbox" ng-model="requiredAction.enabled" ng-change="updateRequiredAction(requiredAction)" id="{{requiredAction.alias}}"></td>
```

Comment:
I don't think it's similar to above. We can end up with overusing IDs if we're not careful. I'm happy with either here, but the ID should use the pattern <id>+<element>. So it would be "{{requiredAction.name}}.enabled" and "{{requiredAction.name}}.defaultAction". I'd still say it would be better to just have ID on the tr and use names within.